src/style.css

```
@@ -0,0 +1,144 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap");
+
+:root {
+  --main-color: #060b35;
+  --main-bg-color: #fff;
+  --serving-color: #616070;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.product-card {
+  height: 406px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: var(--main-bg-color);
+  padding: 16px;
+}
+
+.product-card__image {
+  margin-top: 16px;
+  text-align: center;
+}
+
+.product-card__image > img {
+  width: 160px;
+  height: 134px;
+  object-fit: cover;
+}
+
+.product-card__title {
+  font-size: 12px;
+  font-weight: 500;
+  color: var(--main-color);
+  line-height: 18px;
+  margin: 36px 0 0;
```

Не заставляй інших розробників думати коли можна цього уникнути. Такош радже переробити все на `margin-bottom` і привикнути все так робити. Це позволить в майбутньому уникати margin colapse
```suggestion
  margin-top: 36px;
```